src/hooks/use-suggestions.ts

```diff
@@ -272,12 +272,24 @@ export const useSuggestions = (
                     if (isObject(fieldOf) && !Array.isArray(fieldOf)) {
                         for (const key of Object.keys(fieldOf)) {
                             if (key === '*') continue
+                            if (aliasedKeys.includes(`${matchedField}.${key}`))
+                                continue
                             toAdd.push(`.${key}`)
                         }
                     }
                 } else {
+                    const matchedFieldBeforeDot = matchedField.replace(
+                        new RegExp(`\\.${lastWord}$`),
```

Comment:
The double backslash seems wrong, if you want to escape the `.` you need only `\.`

Reply:
I actually want to escape slash itself, so the resulting string would have \.